src/provider/cloud-element-templates/properties/CustomProperties.js

```diff
@@ -292,7 +292,7 @@ function FeelTextAreaProperty(props) {
         translate = useService('translate');
 
   const TextAreaComponent =
-    isIOMappingProperty(property)
+    !isExternalProperty(property)
```

Comment:
No tests?

Reply:
No tests as we don't have the infrastructure for it. Cf. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/902#issuecomment-1488512815